### DISCUSSION TITLE
change mapping for landing page

### DISCUSSION THREAD

**Discussion by rwijtvliet** [LazyVim/LazyVim#2930] [Q&A] [answered]
When launching neovim without (a) file(s), the landing page is shown. Is there a way to change its mappings? 

I had a look at the mappings [here](http://www.lazyvim.org/keymaps) but couldn't find a plugin responsible for the landing page.

**Comment by dpetka2001** [comment]
The plugin is `dashboard.nvim`. There have been other discussions as well, so you might want to search about it or read the docs of the plugin.

**Reply by rwijtvliet** [reply]
Many thanks! Learning to fish: could I have found this out before asking? ;)

**Reply by dpetka2001** [reply, marked as answer]
Well most plugins regarding UI are in `/lua/plugins/ui.lua`, but if you're not familiar at all with what each plugin does, it's perfectly ok to ask questions like these.
You could do in a file in `~/.config/nvim/lua/plugins/` directory something like
```lua
-- ~/.config/nvim/lua/plugins/ui.lua
return {
  {
    "nvimdev/dashboard-nvim",
    opts = function(_, opts)
      -- stylua: ignore start
      opts.config.center[1] = { action = LazyVim.telescope("files"),                                    desc = " Find File",       icon = " ", key = "t" }
      -- opts.config.center = {
      --   { action = LazyVim.telescope("files"),                                    desc = " Find File",       icon = " ", key = "t" },
      --   { action = "ene | startinsert",                                        desc = " New File",        icon = " ", key = "n" },
      --   { action = "Telescope oldfiles",                                       desc = " Recent Files",    icon = " ", key = "r" },
      --   { action = "Telescope live_grep",                                      desc = " Find Text",       icon = " ", key = "g" },
      --   { action = [[lua LazyVim.telescope.config_files()()]], desc = " Config",          icon = " ", key = "c" },
      --   { action = 'lua require("persistence").load()',                        desc = " Restore Session", icon = " ", key = "s" },
      --   { action = "LazyExtras",                                               desc = " Lazy Extras",     icon = " ", key = "x" },
      --   { action = "Lazy",                                                     desc = " Lazy",            icon = "󰒲 ", key = "l" },
      --   { action = "qa",                                                       desc = " Quit",            icon = " ", key = "q" },
      --   -- stylua: ignore end
      -- }
      for _, button in ipairs(opts.config.center) do
        button.desc = button.desc .. string.rep(" ", 43 - #button.desc)
        button.key_format = "  %s"
      end
    end,
  },
}
```
You can either index directly the component of `opts.config.center` that you want to change the keymap for or you could specify `opts.config.center` as a new table (the lines that are commented out) that will return whatever you want if you want to change more than one keymaps. Of course, you could still index every component separately and do the changes like in the first line that's not commented out. It's up to you what to use.

I don't know how comfortable you are with Lua, but in my personal opinion it would benefit you if you learnt at least some basics to configure things more to your liking.
